src/pages/students/forms/paymentDataForm/PaymentData.tsx

```diff
@@ -1,77 +1,69 @@
 import { MainContainer, 
-        TitleContainer, 
         FormContainer, 
         Label, 
         Input, 
-        Title, 
         Button, 
         ButtonsContainer, 
         Select, 
         SmallInput, 
         InputsContainer,
-        Text} from "../styles/PaymentData.styles";
-import { PaymentPlanName } from "../../../app/types/models/PaymentPlanName";
+        Text} from "./PaymentData.styles";
+import { PaymentPlanName } from "../../../../app/types/models/PaymentPlanName";
 import { useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import * as yup from "yup";
-import { paymentDataScheme } from "../schemes/PaymentData.scheme";
-import { ErrorMessage } from "../../../components/header/ErrorMessage";
-import { useNavigate } from "react-router";
-import { setStudentData } from "../StudentRegistrationFormSlice";
-import type { RootState } from "../../../app/store/store";
-import { useDispatch, useSelector } from "react-redux";
+import { paymentDataScheme } from "./PaymentData.scheme";
+import { ErrorMessage} from "../../../../components/error_message/ErrorMessage"
+import type { PaymentDataProps } from "../../create-student/CreateStudent";
 
+interface FormProp {
+  createStudentCall: (data: PaymentDataProps) => void;
+  paymentData:PaymentDataProps | undefined;
+  stepBack:()=>void;
+}
```

Comment:
Si te fijas, los tres formularios tienen las mismas acciones y reciben informacion, son MUY parecidos, lo que varia es el tipo de dato que recibe.
Lo primero a cambiar, seria que en vez de definir la interfaz en cada una de los formularios, podriamos crear una unica interfaz y que los tres forms usen la misma

Ahora, si los tres van a usar lo mismo, es raro el "createStudentCall" y el "paymentData", porque no en todos los formularios vas a crear el estudiante (solo en el ultimo paso) y no toda la informacion es sobre el pago, en el primer form es sobre los datos del estudiante. Por lo que usaria una interfaz de este estilo


```suggestion
export interface FormProp<T> {
  onNext: (data: T) => void;
  onBack:() => void;
  data: T | undefined;
}
```

Que la separaria en un archivo diferente que se exporte y se pueda utilizar desde los tres formularios